modules/coins.py

```diff
@@ -57,6 +58,7 @@ def __init__(self, bot, events, exports):
             self.send, min_args=2, help="Send coins to a user",
             usage="<nickname> <amount>", authenticated=True)
 
+
```

Comment:
empty space?

Reply:
Spaceman
I always wanted you to go
Into Spaceman (Intergalactic Christ)